client/components/token-field/README.md

```diff
@@ -19,10 +19,14 @@ The `value` property is handled in a manner similar to controlled form component
 ### Properties
 
 - `value` - An array of strings to display as tokens in the field
-- `valueTransform` - Function to call to transform tokens for display.  This is
+- `displayTransform` - Function to call to transform tokens for display.  This is
   used to decode HTML entities embedded in tags - otherwise entities like `&`
   in tag names are double-encoded like `&amp;` (once by the REST API and once
   by React).
+- `saveTransform` - Function to call to transform tokens for saving.  The
+  default is to trim the token value.  This is used to remove leading and
+  trailing spaces from tag names, like wp-admin does.  (Otherwise the REST API
+  won't save them.)
```

Comment:
Might also be worth noting that the `saveTransform` method is called on the search input via `_getMatchingSuggestions`